UPGRADING.md

```diff
@@ -26,3 +26,8 @@ created before the call will now be deleted. In the past this was a problem for
 `dependent: :restrict`.
 
 See [#202](https://github.com/mongoid/mongoid-history/pull/202) for more information.
+
+# Main Changes / Upgrading Notes
+
+* `:all` is just an alias to `:fields` for now, to support existing users of `mongoid-history`. In future, :all
+will track all the fields and relations of trackable class.
```

Comment:
I would make something less apologetic :) 

_Currently, the `:all` option behaves identically to `:fields`. Future versions will track all fields and relations of trackable class when using `:all`._

Also link to an issue/PR that introduced this.